AsaApi/Core/Public/API/ARK/Actor.h

```diff
@@ -6887,6 +6887,7 @@ struct APrimalDinoCharacter : APrimalCharacter
     UE::Math::TVector<double>& SpawnedLocationField() { return *GetNativePointerField<UE::Math::TVector<double>*>(this, "APrimalDinoCharacter.SpawnedLocation"); }
     long double& LastWantsToEnableNavRelevancyField() { return *GetNativePointerField<long double*>(this, "APrimalDinoCharacter.LastWantsToEnableNavRelevancy"); }
     float& DinoArmorDurabilityScalingMultiplierField() { return *GetNativePointerField<float*>(this, "APrimalDinoCharacter.DinoArmorDurabilityScalingMultiplier"); }
+    unsigned __int8& TamedAITargetingRangeField() { return *GetNativePointerField<unsigned __int8*>(this, "APrimalDinoCharacter.TamedAITargetingRange"); }
```

Comment:
Please use `int8_t` instead of `__int8`
https://en.cppreference.com/w/cpp/types/integer